### PR TITLE
Updated levels to reflect standalone plan

### PR DESCRIPTION
**Pull request by jmkulwik**
Originally, we were thinking our console levels would be the Java Lab standalone level type. Now, however, we are planning that Theater levels will be the standalone level type. This adds an example allthethings of what that would look like as well as updates our allthethings console level to more accurately display the student experience. This came up because there was confusion about display bugs related to console levels without instructions - for example, a console level without instructions will have the run button covered up by the "extra links" menu. These bugs do not need fixing as console levels should always have instructions.

Sample standalone level
image

Updated console level
image

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: dashboard/config/scripts/allthethings.script [hunk 1]
```diff
@@ -1,5 +1,6 @@
 hideable_lessons true
 lesson_extras_available true
+published_state 'beta'
```
Comment (molly-moen) [inline dashboard/config/scripts/allthethings.script]: intended change? (same question for the reorder below)

File: dashboard/config/scripts_json/allthethings.script_json [hunk 2]
```diff
@@ -5564,7 +5585,7 @@
     },
     {
       "seeding_key": {
-        "level.key": "ramp_video_artistIntro",
+        "level.key": "2-3 Artist 1 new",
```
Comment (molly-moen) [inline dashboard/config/scripts_json/allthethings.script_json]: intended changes?

File: dashboard/config/scripts_json/allthethings.script_json [hunk 3]
```diff
@@ -6179,4 +6211,4 @@
   "lessons_opportunity_standards": [
 
   ]
-}
+}
```
Comment (molly-moen) [inline dashboard/config/scripts_json/allthethings.script_json]: nit: add blank line

**Review by molly-moen — COMMENTED** [review]
Two questions:
1. This doesn't change what a standalone level is yet, right? It's just allthethings examples?
2. Is there a slack thread somewhere discussing why we are making this change?

**Comment by moneppo** [comment]
> 2. Is there a slack thread somewhere discussing why we are making this change?

No - this deserves a write up but I'll put some notes here for now. For standalone projects, I want to focus on theater to start as it has the most potential for open-ended student projects. The other mini-app types don't make a ton of sense for independent projects except for Playground which we don't have good data on how it will perform/cost. So, to scope down what we need to support/think about right now, I think it makes sense to just keep our example standalone to be a theater project.

**Comment by jmkulwik** [comment]
> 1. This doesn't change what a standalone level is yet, right? It's just allthethings examples?

That's a good point. I'll go ahead and update the standalone level so everything's in sync.

File: dashboard/config/scripts/allthethings.script (hunk 1 above)
Comment (jmkulwik) [reply dashboard/config/scripts/allthethings.script]: Huh I actually don't know. I edited the script through the UI and saved it - does this mean I should have seeded first?

Comment (jmkulwik) [reply dashboard/config/scripts/allthethings.script]: Edit: I went back and seeded first and got similar changes.

Comment (jmkulwik) [reply dashboard/config/scripts/allthethings.script]: Asked in #dashboard-dev and this was the response:
"these are mostly harmless -- when we update the format that these files get serialized into, we don't always go back and update all the files to match. however for your own sanity what I'd recommend is including only the bare minimum you need to accomplish your change, to minimize merge conflicts. in this case, only the .script file does anything, so you can revert the entire .script_json file."

^I'll go ahead and take that recommendation to remove the unneeded changes

**Review by molly-moen — APPROVED** [review]

**Pull request merged**